packages/common/src/functions-api.ts

```diff
@@ -80,6 +80,11 @@ export interface CancelAppointmentRequest {
   appointmentId: string;
 }
 
+export const ConfirmAppointmentFunctionName = "confirmedAppointment";
+export interface ConfirmAppointmentRequest {
+  appointmentId: string;
+}
+
```

Comment:
Maybe we should return the confirmed appointment to the client?
So it would show it as confirmed?